content/getting-started/usage.md

```diff
@@ -233,11 +234,11 @@ By using Hugo's server in production, you are able to deploy just the source fil
 
 Interested? Here are some great tutorials contributed by Hugo users:
 
-* [hugo, syncthing](http://fredix.xyz/2014/10/hugo-syncthing/) (French) by Frédéric Logier (@fredix)
+* [hugo, syncthing](https://fredix.xyz/2014/10/hugo-syncthing/) (French) by Frédéric Logier (@fredix)
```

Comment:
@digitalcraftsman Do you think we should even keep this section on using `hugo` server in production? I started with Hugo 18 months ago, and since then, I've always heard that `hugo server` is a dev-only feature. Is this something that the team really wants to take on in terms of support?